packages/terra-clinical-result/src/_BloodPressureDisplay.jsx

```diff
@@ -46,7 +56,20 @@ const BloodPressureDisplay = ({
     <ConditionalWrapper
       key={`del-${type}-${result.eventId}`}
       condition={result.statusInError}
-      wrapper={children => <del>{children}</del>}
+      wrapper={children => (
+        <span>
+          <s aria-hidden="true">{children}</s>
+          {isValidValue
+            ? (
+              <VisuallyHiddenText text={intl.formatMessage(
+                { id: 'Terra.clinicalResult.statusInErrorStrikethrough' },
+                { strikethroughResult: `${result.result.value} ${result.result.unit ? result.result.unit : ''}` },
```

Comment:
Just curious, if there is no unit for the result, will the screen reader read out anything else after the value? (Basically want to confirm if the empty string would cause any unexpected behavior or not)

Reply:
Great question, the space doesn't cause any issues - however I will mention that I am currently adding a check here for if hideUnit is true - we won't want to read out the unit if it's not being show.